spec/client_spec.rb

```diff
@@ -26,11 +27,9 @@ def coda_entities
       'columns',
       'controls',
       'docs',
-      'folders',
       'formulas',
       'miscellaneous',
       'rows',
-      'sections',
```

Comment:
Are these two entities just not relevant anymore with the update?

Reply:
Dez and I took a look here and it seems that this list is used to assert entity classes exist. The deletions look good as they do not exist but there are a couple that are missing if we want to be consistent, namely `categories` and `pages`. Honestly this test seems super clunky and awkward, I'd be cool with updating it to be fully consistent or removing it.